### PR TITLE
PlebNetDev_Meeting_Notes_Jul2023

### PR DESCRIPTION
**Pull request by Bitc0indad**
PlebNet Dev Monthly Meetup in July 

Meeting Minutes attached for review @eriknewland  & @bitkarrot  please add when I missed specifics thanks!

### CODE REVIEW

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#### 5. Other Discussion Points
+
+- Printify e-commerce and automated bot trading.
+- Reconciliation issues for producers accepting bitcoin lightning sales.
+- Randy's decentralized GPG key project for signing commits.
+- More unit tests and modules for LNBits, aiming for release in August.
+
+## Action Items:
+
+1. Continue working on NOSTR Projects (Spellkit, NOSTR accounts, NIP\*05, and front-end application). (ErikNewland)
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:60]: nostrkit, not spell kit

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Date: July 29th, 2023 @11pm EST
+
+## Attendees: Bitkarrot, BitcoinDad, ErikNewland, SurferSol, Tony
+
+### Agenda
+
+#### 1. NOSTR / PYTHON Projects Update
+
+- Progress on spellkit template outline.
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:13]: nostr kit not spell kit

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Development of NOSTR accounts initial boilerplate.
+- NIP\*05 issues: Python attempt to vanity npub (RONNA).
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:15]: Rana not RONNA

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- Public folder with .wellknown for NOSTR lighting addresses.
+- Development of one-click clone template for NOSTR domains.
+- Get an aldy account with lnurl.
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:18]: getalby.com not aldy

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+- Plans for Erik to spin up the DEV server.
+- Front-end application development allowing creation of entries.
+- Bitcoindad finishing up LPTHW Ex. 50 Build your website. Will provide Coingecko/Kraken API programs for integration into website.
+- Meetup with new plebnet devs like surfersol help establish coding enviroment troubleshooting any setup issues. 
+
+#### 2. NOSTR Project Ideas
+
+- NOSTR Telegram Bridge for public messaging and encouraging NOSTR adoption.
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:26]: bloggingbitcoin is working on these nostr bridges

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- NOSTR Discord Bridge to maintain a public presence and active channel.
+- Integration of ASTRO template for automatic NOSTR posting when a .md blog post is created.
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:28]: the astro-nostr project is an open for anyone interested in working on something independently

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+#### 3. Plebnet.dev Updates - Bitkarrot
+
+- Establishing testnet core lighting node and lnbits instance.
+- Plans for scalable growth including bootstrapping.
+- Post-September membership fees and sponsorship, merchandise store for funding.
+- Open source ecommerce store development for plebnet.dev.
+- Plebnet.dev's business model discussion: unlimited access or minor fee.
+- No public instance?
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:37]: for now, no public testnet. members only

File: PlebNetDev_Meeting_Notes_Jul2023.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+- Importance of understanding the fundamentals of how checkout works with payment systems.
+- Integrated lightning test net for Zebedee initiative.
+- "Good First Issues" meeting with other Senior Devs proposed.
+
+#### 5. Other Discussion Points
+
+- Printify e-commerce and automated bot trading.
+- Reconciliation issues for producers accepting bitcoin lightning sales.
+- Randy's decentralized GPG key project for signing commits.
+- More unit tests and modules for LNBits, aiming for release in August.
```
Comment (bitkarrot) [inline PlebNetDev_Meeting_Notes_Jul2023.md:56]: Please edit this line to say the following: 

The pylnbits sdk release will be in august. If anyone is interested in writing docs or helping with unit testing, this is a also a good first project to work on.

**Review by bitkarrot — CHANGES_REQUESTED** [review]
please see corrections requested. Thanks for posting!

**Comment by Bitc0indad** [comment]
@bitkarrot Feedback updated. @eriknewland  any updates please comment

**Comment by bitkarrot** [comment]
@Bitc0indad looks fine to me, approved. @eriknewland  for the final confirm and merge please

**Review by eriknewland — APPROVED** [review]
looks good!

**Pull request merged**